homeworks/HW_05/task1.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+let averageGrade = 102;
+
+if (averageGrade <= 60) {
+    console.log('Незадовільно');
+}   else if (averageGrade >= 60 && averageGrade <= 70) {
```

Comment:
Умова averageGrade >= 60 тут не потрібна - коли не справцював перший іф, вже тоді ми можемо бути впевнені, що вона виконується
```suggestion
}   else if ( averageGrade <= 70) {
```
І теж саме для наступних умов (averageGrade >=71, averageGrade >= 81, averageGrade >= 91)